### PR TITLE
Use local workbox libraries

### PR DESCRIPTION
**Pull request by MikeyBeLike**
cc @XhmikosR 

https://github.com/twbs/bootstrap/issues/26600

I couldn't get tests to pass locally, dunno why?

### CODE REVIEW

**Comment by XhmikosR** [comment]
Thanks! I'm gonna play with this as soon as I have some time and let you know.

**Comment by Lyricalz** [comment]
I can't get this to run locally against 127.0.0.1, so can't really test the service worker, ``docs-serve"`` loads 0.0.0.0 or some sort

**Comment by Lyricalz** [comment]
I created an ODE here: https://load-workbox-locally--bootstrap-sw.netlify.com/

If you wanna have a quick look

File: sw.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 /* global workbox:false */
 
-self.importScripts('assets/js/vendor/{fileName}')
+self.importScripts('assets/js/vendor/workbox/{workboxVersion}/workbox-sw.js')
+
+workbox.setConfig({
+  modulePathPrefix: '/assets/js/vendor/workbox/{workboxVersion}/'
```
Comment (XhmikosR) [inline sw.js]: This will fail with our current live docs structure. Can you make it relative?

Comment (Lyricalz) [reply sw.js]: It is relative to the file, no?

Comment (XhmikosR) [reply sw.js]: nope, it's relative to the server root :)

**Comment by XhmikosR** [comment]
@Lyricalz: I'm getting errors on your preview site `ServiceWorker registration failed:  TypeError: "ServiceWorker script at https://load-workbox-locally--bootstrap-sw.netlify.com/sw.js for scope https://load-workbox-locally--bootstrap-sw.netlify.com/ encountered an error during installation."`

**Comment by XhmikosR** [comment]
But anyway, I'm gonna play with this later. I was planning to switch to `generateSW()` anyway, so this is a good time to look into it. I just need to find some time :)

**Comment by Lyricalz** [comment]
@XhmikosR are you by chance using firefox? I'm seeing a lot of those errors in google related to firefox.

**Comment by Lyricalz** [comment]
@XhmikosR did you manage to check this on another browser to see if you still had this same error as above?

**Comment by XhmikosR** [comment]
@Lyricalz: sorry, I hadn't have the time to look into this. I haven't forgotten about it, it's in my TODO list.

**Pull request closed without merging**